ci/builders/mac_unopt.json

```diff
@@ -125,17 +125,20 @@
                 "use_rbe": true
             },
             "gn": [
-               "--runtime-mode",
-               "debug",
-               "--unoptimized",
-               "--no-lto",
-               "--prebuilt-dart-sdk",
-               "--force-mac-arm64",
-               "--mac-cpu",
-               "arm64",
-               "--rbe",
-               "--no-goma",
-               "--xcode-symlinks"
+                "--runtime-mode",
+                "debug",
+                "--unoptimized",
+                "--no-lto",
+                "--prebuilt-dart-sdk",
+                "--force-mac-arm64",
+                "--enable-impeller-vulkan",
+                "--enable-impeller-opengles",
```

Comment:
We don't have to do this here, but unless there are issues I'm not aware of we should remove these flags. its too dangerous that we have a "by default don't test our impeller backends" setting.